pkg/virt-launcher/virtwrap/api/testdata/domain_arm64.xml.tmpl

```diff
@@ -66,6 +66,7 @@
     </kvm>
     <pvspinlock state="off"></pvspinlock>
     <pmu state="off"></pmu>
+    <vmport state="off"></vmport>
```

Comment:
@ebblake I think changes in all file in this directory are needed too: `pkg/virt-launcher/virtwrap/converter/testdata/`